src/hooks/types.tsx

```diff
@@ -0,0 +1,49 @@
+export type UseParamsOptions = Readonly<{
```

Comment:
types should be in a folder called `types` inside `src`. Something like `types/entities.d.ts` for example.